packages/react-magma-dom/src/components/Dropdown/DropdownExpandableMenuButton.tsx

```diff
@@ -86,6 +86,7 @@ export const DropdownExpandableMenuButton = React.forwardRef<
         expandableMenuGroupContext.expandableMenuButtonHasIcon
       }
       isInverse={context.isInverse}
+      style={expandableMenuItemContext.disabled ? { background: 'none' } : {}}
```

Comment:
Instead of making this inline, it would be ideal to add to `StyledAccordionButton`. That way if users want to add extra styles, they can.